### ISSUE TITLE
Regression: Exception during build of previously working sandbox

### ISSUE THREAD

**Issue by jtheisen** [CompuIves/codesandbox-client#1727]
# 🐛 bug report

https://codesandbox.io/s/ll1wmkpqoq

The above sandbox fails with the message

> Could not find module in path: '@babel/runtime/helpers/interopRequireWildcard' relative to '/src/index.tsx'

It used to work previously, it strangely stopped working when I made a clone of it and the returned. Now both the clone and the original don't work any longer.

**Comment by JonShort** [comment]
I received the same error you mention above when I opened the sandbox you linked 🤔 

I managed to fix it locally (and don't seem to be able to reproduce it now) by:

1). Remove the decorator on line 31 (`@observer`)
2). Allow the app to recompile
3). Add the decorator back

It could be an erroneous babel compile was cached? It's a strange case, although I do remember seeing the babel error before in my sandboxes (i think it was when I tried to use syntax which wasn't supported by my babel config)

Hope this helps 👍
